plugins/versionpress/src/Cli/VPCommandUtils.php

```diff
@@ -23,6 +23,9 @@ public static function runWpCliCommand($command, $subcommand, $args = [], $cwd =
             $args['color'] = null;
         }
 
+        // For commands that were run under root - #1049
+        $args['--allow-root'] = null;
```

Comment:
This was actually not correct, fixed in https://github.com/versionpress/versionpress/pull/1272/commits/ceca06332d2d2ca6f8cfd6999e5d5526542f210c.